core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -577,6 +579,7 @@ export namespace RealityModelTileTree {
     protected _classifier?: SpatialClassifierTileTreeReference;
     protected _mapDrapeTree?: TileTreeReference;
     protected _getDisplaySettings: () => RealityModelDisplaySettings;
+    protected _contextRealityModel?: ContextRealityModel;

```

Comment:
@pmconne are you ok having optional `ContextRealityModel` object on  `RealityModelTileTree.ReferenceBaseProps` ?

Reply:
Strictly for accessing the `invisible` flag? Would prefer looser coupling like `protected _isInvisible?: () => boolean`.

Reply:
@pmconne updated